vtn/pom.xml

```diff
@@ -10,38 +10,40 @@
     <packaging>jar</packaging>
 
     <properties>
+
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
+        <junit.version>4.12</junit.version>
+        <slf4j.api.version>1.7.25</slf4j.api.version>
         <maven.plugin.version>3.0.2</maven.plugin.version>
     </properties>
-
 <dependencies>
 
-    <!-- https://mvnrepository.com/artifact/org.projectlombok/lombok -->
-    <dependency>
-        <groupId>org.projectlombok</groupId>
-        <artifactId>lombok</artifactId>
-        <version>1.16.18</version>
-        <scope>provided</scope>
-    </dependency>
+        <dependency>
+            <groupId>com.squareup.retrofit2</groupId>
+            <artifactId>retrofit</artifactId>
+            <version>RELEASE</version>
```

Comment:
What are the benefits of using RELEASE? If a new release is released then how sure are you ki the current implementation is not depreciated or updated causing the application to not compile?